configs/bundle_config.yaml

```diff
@@ -348,3 +348,13 @@ databundles:
     output: [cutouts/cutout-2013-era5.nc]
     disable_by_opt:
       build_cutout: [all]
+
+  bundle_cutouts_westernasia:
```

Comment:
Amazing @Emre-Yorat89 :D
This seems to be ok, the only comment I'd recommend is to add a comment on top of the bundle similarly to the others.
You may use Asia or Africa as templates.